### PR TITLE
[SDK-2302] Update .Net Core Web API quickstart

### PR DESCRIPTION
**Pull request by stevehobbsdev**
- Rename .Net Core Web API qs + other small fixes
- Add a link to application settings to API tutorial

### CODE REVIEW

File: articles/quickstart/backend/aspnet-core-webapi-2/03-troubleshooting.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ You can inspect a JWT with the [JWT.io](https://jwt.io/) website. Use the debugg
 
 The screenshot below shows the following information:
 * The token is signed with the RS256 algorithm
-* The issuer of the token is https://jerrie.auth0.com/
-* The audience of the token is https://rs256.test.api
+* The issuer of the token is `https://jerrie.auth0.com/`
```
Comment (frederikprijck) [inline articles/quickstart/backend/aspnet-core-webapi-2/03-troubleshooting.md]: I do not want to block the PR om this, but want to ask ...  What do we do with these kind of links, given they are not part of the SDK anymore ? I know these are merely here as a matter of explanation but should we consider updating this to something else, eventually? This probably has implications on the samples as well tho.

File: articles/quickstart/backend/aspnet-core-webapi/01-authorization.md [hunk 2]
```diff
@@ -13,10 +12,11 @@ github:
 contentType: tutorial
 useCase: quickstart
 ---
+<!-- markdownlint-disable MD041 MD002 -->
 
 <%= include('../../../_includes/_api_auth_intro') %>
 
-<%= include('../_includes/_api_create_new', { sampleLink: 'https://github.com/auth0-samples/auth0-aspnetcore-webapi-samples/tree/master/Samples/hs256' }) %>
+<%= include('../_includes/_api_create_new') %>
```
Comment (frederikprijck) [inline articles/quickstart/backend/aspnet-core-webapi/01-authorization.md:19]: I think we should also drop this SampleLink for the 2.1 sample. As well as drop the sample.

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: articles/quickstart/backend/aspnet-core-webapi/01-authorization.md (hunk 2 above)
Comment (stevehobbsdev) [reply articles/quickstart/backend/aspnet-core-webapi/01-authorization.md:19]: Samples were dropped in this commit: https://github.com/auth0-samples/auth0-aspnetcore-webapi-samples/commit/4e9b146e7693213b5452fbd2c9c13e144c67eca9

File: articles/quickstart/backend/aspnet-core-webapi-2/03-troubleshooting.md (hunk 1 above)
Comment (stevehobbsdev) [reply articles/quickstart/backend/aspnet-core-webapi-2/03-troubleshooting.md]: I've renamed the issuer value to `https://example.auth0.com` throughout.

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**